packages/fixed-decimal/src/currencies.ts

```diff
@@ -0,0 +1,67 @@
+import { FixedDecimal } from './FixedDecimal'
+
+/**
+ * Creates a {@link FixedDecimal} constructor for a currency.
+ * @param decimalPlaces the number of decimal places for the currency
+ */
+const createTokenConstructor =
+  <T extends FixedDecimal>(
+    decimalPlaces: ConstructorParameters<typeof FixedDecimal>[1]
+  ) =>
+  (value: ConstructorParameters<typeof FixedDecimal>[0]): T =>
+    new FixedDecimal(value, decimalPlaces) as T
+
+/**
+ * A {@link FixedDecimal} representing an amount of Ethereum ERC-20
+ * AUDIO tokens, which have 18 decimal places.
+ *
+ * Used on the protocol dashboard and in the governance and staking systems.
+ * Also used for balance totals after adding linked wallets on the Rewards page.
+ */
+type AudioTokens = FixedDecimal & { _brand: 'AUDIO' }
```

Comment:
nit: Could probably have made a type utility for this?
`type Branded<T, Brand extends string> = T & { _brand: Brand }`

Just for the sake of consistency and/or making it easier to brand constructed currencies.
Maybe I'm too obsessed with clever typing...

Reply:
I'm also wondering whether it would make sense for FixedDecimal to support brand as a first-class field (optional). It would certainly make runtime usage a little more explicit as we could check and throw if converting between values and inspecting it in dev tools would show the brand as part of the object.

Reply:
I'm considering changing this to brand the underlying `bigint` and have a type parameter to `FixedDecimal` - i'll take a look and see what that's like

Reply:
done! https://github.com/AudiusProject/audius-protocol/pull/6724/files